src/cpp/b2w.cpp

```diff
@@ -243,8 +243,11 @@ int main(int argc, char* argv[])
                     rLen[j] = 0;
                 }
                 tmp.rLen = rLen;
-                sprintf(filename, "w-%s-%d-%d.reads.fas",  // read window filename
+                sprintf(filename, "w-%s-%u-%u.reads.fas",  // read window filename
```

Comment:
Names generated in C++ code here.

Reply:
But if we change in python code, we could just do `"w-%s-%u-%u-reads.fasta"` here, right?

Reply:
but we will need to make sure that [a future](https://github.com/cbg-ethz/shorah/blob/b17422e0f45b3ee53bc5882d0ca2baa3e506ba7f/src/cpp/b2w.cpp#L247)  `sanitize_id(tmp.in->header->target_name[i])`  does the exact same thing as [in python](https://github.com/cbg-ethz/shorah/blob/b17422e0f45b3ee53bc5882d0ca2baa3e506ba7f/src/shorah/amplicon.py#L326) to keep thing coherent within the whole pipeline.